python/foglamp/services/south/modify_process.py

```diff
@@ -0,0 +1,135 @@
+                                  delete_url)
+                    raise StorageServerError(code=resp.status, reason=resp.reason, error=jdoc)
+    except StorageServerError as ex:
+        _LOGGER.warning('StorageServerError raised while unregistering service %s with id %s', service_name, microservice_id)
+    else:
+        _LOGGER.warning('Unregistered service %s: %s', service_name, jdoc)
+
+
+async def modify_process_name(service_name, core_management_host, core_management_port):
+    _LOGGER.warning('Getting schedule id for %s', service_name)
+    # Get schedule id
+    storage = StorageClientAsync(core_management_host, core_management_port)
+
+    try:
+        payload = PayloadBuilder().SELECT("id").WHERE(['schedule_name', '=', service_name]).payload()
+        result = await storage.query_tbl_with_payload('schedules', payload)
+    except Exception:
+        raise
+
+    if int(result['count']):
+        sch_id = result['rows'][0]['id']
+    else:
+        _LOGGER.warning('No schedule id found for %s. Exiting...', service_name)
+
+    # Modify process name
+    try:
+        put_url = "{}://{}:{}/foglamp/schedule/{}".format(_HTTP, _HOST, _PORT, sch_id)
+        data = '{"process_name": "south_c"}'
+        async with aiohttp.ClientSession() as session:
+            async with session.put(put_url, data=data) as resp:
+                status_code = resp.status
+                jdoc = await resp.text()
+                if status_code not in range(200, 209):
+                    _LOGGER.error("Error code: %d, reason: %s, details: %s, url: %s", resp.status, resp.reason, jdoc,
+                                  put_url)
+                    raise StorageServerError(code=resp.status, reason=resp.reason, error=jdoc)
+    except Exception:
+        raise
+    _LOGGER.warning('Modified process_name from "south" to "south_c" for Python South Plugin [%s]: %s', service_name, jdoc)
+
+
+async def reenable_service(service_name):
+    # Re-enable schedule
+    try:
+        put_url = "{}://{}:{}/foglamp/schedule/enable".format(_HTTP, _HOST, _PORT)
+        data = '{"schedule_name": "%s"}' % service_name
+        async with aiohttp.ClientSession() as session:
+            async with session.put(put_url, data=data) as resp:
+                status_code = resp.status
+                jdoc = await resp.json()
+                if status_code not in range(200, 209):
+                    _LOGGER.error("Error code: %d, reason: %s, details: %s, url: %s", resp.status, resp.reason, jdoc,
+                                  put_url)
+                    raise StorageServerError(code=resp.status, reason=resp.reason, error=jdoc)
+    except Exception:
+        raise
+    _LOGGER.warning('Enabled Python South Plugin [%s] after changing process_name to south_c', service_name)
+
+
+async def change_to_south_c(service_name, microservice_management_host, core_management_host, core_management_port, microservice_id):
+    global _HTTP, _HOST, _PORT
```

Comment:
Can we avoid globals

Reply:
global declaration is required as we intend to modify the values, otherwise for reading only, no global declaration is needed.